fixtures/wrangler-dev-api-app/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "wrangler-dev-api-app",
+	"version": "1.0.0",
+	"description": "",
+	"scripts": {
+		"dev": "node --no-warnings ./src/wrangler-dev.mjs"
```

Comment:
Now we have `worker.stop()` we could easily add a unit test here that is very similar to wrangler-dev.mjs.

Reply:
Something along the lines of:

```ts
import undici from "undici";
import wrangler from "wrangler";

describe("worker", () => {
	it("should invoke the worker and exit", async () => {
		//since the script is invoked from the directory above, need to specify index.js is in src/
		const worker = await wrangler.unstable_dev("src/index.js");

		const resp = await undici.fetch("http://localhost:8787/");
		const text = await resp.text();

		expect(text).toMatchInlineSnapshot(
			`"http://localhost:8787/ Fri Jul 01 2022 11:41:21 GMT+0100 (British Summer Time)"`
		);

		await worker.stop();
	});
});
```